src/server/controllers/Smoothie/SmoothieController.js

```diff
@@ -455,6 +455,20 @@ class SmoothieController {
             this.emit('serialport:read', res.raw);
         });
 
+        // handle action:xxx host commands
+        this.runner.on('action', (res) => {
+            log.error(`incoming action: action:${res.message} !`);
+            if (res.message === 'pause') {
```

Comment:
This would be better expressed as a switch statement.  The else if formulation is of course semantically equivalent, but the switch formulation makes it clear and explicit that the choices are based only on the value of a single variable.